Huion_G10T/index.md

```diff
@@ -0,0 +1,20 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: G10T
+image: Huion_G10T
+report_rate: 233
+working_area:
+    width: 10
+    height: 6.25
+    resolution: 5080
+pen:
+    pressure_levels: 8192
+frame_controls: 6 Press Keys
```

Comment:
This should contain the same description as the one for GC610. Actually, they look remarkably similar, can you check if the parameters, USB descriptors and HID report descriptors match? If they do, then perhaps it's the same tablet under a different name.

Reply:
Sorry, I do not have the description when I review the issue in digimend. I will remove it. I think you have this description.

Reply:
I see it has difference. In particular,  Huion_G10T has  `bcdHID  1.0b`. And Huion_GC610 has `bcdHID               1.11`

Reply:
Ah, OK, thanks for checking. However, please fix the description of frame controls.